:Title: Research
:URL:
:save_as: index.html

.. image:: /images/bl_logo.png
    :alt: Beatson Lab Group logo

The Beatson group aims to better understand the molecular mechanisms of
infectious disease and identify potential therapeutic and diagnostic targets by
exploiting “Next-gen” genomic data. A major focus of the group is the
comparative analysis of genomes obtained from local clinical isolates of
important human pathogens such as *Escherichia coli, Pseudomonas aeruginosa,
Staphylococci, Streptococci, Legionella pneumophila* and *Acinetobacter
baumannii*. In particular, the group is interested in the evolution and
mobility of genes encoding virulence factors that are widely conserved amongst
bacterial pathogens (e.g. fimbriae, pili and type III and type IV secretion
systems and secreted effectors). World-wide there are thousands of bacterial
genome and meta-genome projects in progress, but the bottle-neck remains data
analysis. A major priority in the group is the development of bioinformatics
software to effectively utilize Next-gen sequence data.

Current projects range from researching fundamental questions such as “how has
*Escherichia coli* pathogenesis evolved?” and "how are genes transferred 
between bacteria?" to the identification of potential diagnostic and vaccine 
targets and the development of software tools to enable integrated analysis of 
100’s or 1000’s of bacterial genomes. 


Recent research highlights
--------------------------

**NHMRC Funding**

Congratulations to Dr Scott Beatson on being awarded a Level 2 NHMRC Career 
Development Fellowship. The project was titled "Bacterial Pathogenomics: 
whole-genome sequencing to investigate infection transmission, pathogenesis 
and antibiotic resistance". A total of $455 452 was awarded and will begin 
in 2015.


**Complete genome of representative ST131 strain**

We determined the complete genome sequence of Escherichia coli EC958: a high 
quality reference sequence for the globally disseminated multidrug resistant 
E. coli O25b:H4-ST131 clone. We used a 3rd generation sequencing technology, 
PacBio SMRT sequencing to determine this genome and associated plasmids. 
We declare that "Large discrepancies between ST131 genomes are likely due to 
misassembly of E. coli NA114". [Link_] 


**Escherichia coli ST131**

In April 2014 we had a large scale study of *E coli* ST131 published_ in PNAS. 
We wrote a `blog post`_ about the importance of this study. Both `UQ News`_ 
and `7 News`_ presented a news articles about this study. The data is 
available on GitHub_.

.. _published: http://www.pnas.org/content/early/2014/03/28/1322678111.abstract
.. _`blog post`: http://beatsonlab.com/PNAS_ST131_2014.html
.. _`UQ News`: http://www.uq.edu.au/news/article/2014/04/evolving-superbug-threatens-create-infection-tsunami
.. _`7 News`: https://au.news.yahoo.com/video/watch/22383656/researchers-declare-war-on-superbugs/
.. _`GitHub`: https://github.com/BeatsonLab-MicrobialGenomics/ST131_99
.. _Link: http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0104400


Collaborations
--------------

**National:** Prof Mark Schembri (UQ), Prof Mark Walker (UQ), A/Prof Andrew 
Barnes (UQ), Dr Mark Turner (UQ), A/Prof Glen Ulett (Griffith), Dr Kari Gobius 
(CSIRO), Prof David Paterson (UQ), A/Prof Scott Bell (QHealth), A/Prof Claire 
Wainwright (QHealth), Prof Elizabeth Hartland (UMel).  

**International:** Dr Mat Upton (Manchester), Prof David Gally (Edinburgh), 
Dr Andrew Roe (Glasgow), Prof Tim Walsh (Cardiff), 
Prof Ian Henderson (Birmingham).

**Read more about our research in our** blog_.

**Have a look at some** photos_ **of recent activities**.

.. _blog: http://beatsonlab-microbialgenomics.github.io/archives
.. _photos: http://beatsonlab-microbialgenomics.github.io/photos.html


Available projects
------------------

Potential projects include:
    * Developing software for Next-Gen bacterial genome and meta-genome 
      assembly, annotation and analysis
    * Investigating the role of positive selection in the evolution of 
      bacterial pathogenesis
    * Genomic epidemiology: investigating outbreaks using genome sequencing
    * Reverse vaccinology: identifying candidate bacterial vaccine candidates
    * Investigating the role of bacteriophage in mobilizing virulence genes

