Projects
========

**Microbial genomics**
We aim to exploit next-generation DNA sequencing to better understand how medically important bacteria cause disease, become resistant to antibiotics and disseminate globally. Our major focus is the role of mobile genetic elements in the evolution of virulence and antimicrobial resistance amongst pathogens such as *Escherichia coli*, *Streptococcus pyogenes*,  and *Pseudomonas aeruginosa*. Our group develops new computational approaches for the rapid analysis of bacterial genome data from the latest sequencing technologies. 


**Project 1: Genomic investigation of antimicrobial resistance.**
Carbapenem resistant *Enterobacteriaceae* (CRE) pose an urgent risk to global human health. CRE, which include organisms such as *Klebsiella pneumoniae* and *Escherichia coli*, are resistant to almost all currently available antibiotics. Almost 50% of patients who develop bloodstream infections with these organisms die from the infection. In this project we use Pacific Biosciences Single Molecule Real-Time (SMRT) sequencing to determine the complete genomes of CRE isolated from local hospitals. Comparative genomic analysis will enable us to recognise the genetic cause of antibiotic resistance and the relationship between isolates from an outbreak. As we outlined recently (Beatson SA, Walker MJ (2014) Science 345: 1454-1455), in contrast to short-read sequencing technologies commonly used in genomic studies, SMRT sequencing allows complex antimicrobial resistance elements to be properly characterised.


**Project 2: Phylogenomic analysis of global pandemic E. coli.**
*Escherichia coli* sequence type 131 (ST131) is a globally disseminated, multidrug resistant clone responsible for a high proportion of urinary tract and bloodstream infections. By sequencing the genomes of ST131 isolates from all over the world we were able to use phylogenomic analysis to investigate the rapid emergence and successful spread of this clone (Petty, Ben Zakour et al., PNAS, 2014). In this project we will analyse the genomes of other global pandemic multidrug resistant *E. coli* clones in order to understand of the role of mobile genetic elements and antibiotic resistance in their evolution.


**Project 3: Novel methods to visualise bacterial genomic data.**
We have previously developed easy-to-use software such as BRIG and Easyfig for visualising bacterial genome comparisons.
We are seeking computationally focused students to develop novel web-based software for intuitive visualization and reporting of virulence and antimicrobial resistance gene profiles from genome data generated using Illumina or Nanopore technologies.


**Techniques you learn in our group may include:** Comparative genomic analysis, Phylogenomics, Bioinformatics, Python programming, Web programming; Joint projects that have both computational and “wet-lab” components may be arranged with other SCMB microbiology group leaders on request.


Please contact Scott directly (scott dot beatson at uq.edu.au) for further information on opportunities for Undergraduate, Masters, Honours and PhD projects.
